sdk/core/azure-core-amqp/cgmanifest.json

```diff
@@ -6,7 +6,7 @@
         "Type": "git",
         "git": {
           "RepositoryUrl": "https://github.com/Azure/azure-uamqp-c",
-          "CommitHash": "b7bdf307a0db8840156d9f6e40f704c1d5924306"
```

Comment:
This is no longer a dependency (because we have a vendored copy in our repo). We need to add umock_c, azure_macro_utils_c, and azure_c_shared_utility

Reply:
Thanks! Done. I took the commit hashes of what is the current version in vcpkg.
BTW, I think it could be possible to trim the umock dependency. We can do it later.